cmd/storage-rest-server.go

```diff
@@ -951,42 +951,43 @@ func streamHTTPResponse(w http.ResponseWriter) *httpStreamResponse {
 	blockCh := make(chan []byte)
 	h := httpStreamResponse{done: doneCh, block: blockCh}
 	go func() {
-		defer close(doneCh)
```

Comment:
do we not want to close doneCh here?

Reply:
Yeah, but adding it explicit at the return.

Reply:
Adding it to the return, so it is a bit more obvious that it cannot return from anywhere else.